tests/acceptance/01_vars/02_functions/not.cf

```diff
@@ -0,0 +1,98 @@
+######################################################
+#
+#  Test that not() behaves as expected
+#
+#####################################################
+body common control
+{
+    inputs => { "../../default.cf.sub" };
+    bundlesequence  => { default("$(this.promise_filename)") };
+    version => "1.0";
+}
+
+#######################################################
+
+bundle agent init
```

Comment:
Unless you are actually initing something, you don't need this empty bundle.

Reply:
Thanks, removed.